src/test/TestGhoStewards.t.sol

```diff
@@ -40,45 +65,231 @@ contract TestGhoStewards is Test {
       ),
       true
     );
+
+    assertEq(
+      IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL).getRateLimitAdmin(),
+      address(GHO_CCIP_STEWARD)
+    );
+    assertEq(
+      IUpgradeableLockReleaseTokenPool(GHO_TOKEN_POOL).getBridgeLimitAdmin(),
+      address(GHO_CCIP_STEWARD)
+    );
+
+    assertEq(
+      Gsm(GHO_GSM_USDC).hasRole(Gsm(GHO_GSM_USDC).CONFIGURATOR_ROLE(), address(GHO_GSM_STEWARD)),
+      true
+    );
+    assertEq(
+      Gsm(GHO_GSM_USDT).hasRole(Gsm(GHO_GSM_USDT).CONFIGURATOR_ROLE(), address(GHO_GSM_STEWARD)),
+      true
+    );
+  }
+
+  function testGhoAaveStewardUpdateGhoBorrowCap() public {
+    uint256 currentBorrowCap = _getGhoBorrowCap();
+    uint256 newBorrowCap = currentBorrowCap + 1;
+    vm.prank(RISK_COUNCIL);
+    GHO_AAVE_STEWARD.updateGhoBorrowCap(newBorrowCap);
+    assertEq(_getGhoBorrowCap(), newBorrowCap);
+  }
+
+  function testGhoAaveStewardUpdateGhoSupplyCap() public {
+    uint256 currentSupplyCap = _getGhoSupplyCap();
+    assertEq(currentSupplyCap, 0);
+    uint256 newSupplyCap = currentSupplyCap + 1;
+    // Can't update supply cap even by 1 since it's 0, and 100% of 0 is 0
+    vm.expectRevert('INVALID_SUPPLY_CAP_UPDATE');
+    vm.prank(RISK_COUNCIL);
+    GHO_AAVE_STEWARD.updateGhoSupplyCap(newSupplyCap);
+  }
+
+  function testGhoAaveStewardUpdateGhoBorrowRate() public {
+    IDefaultInterestRateStrategyV2.InterestRateData memory currentRates = _getGhoBorrowRates();
+    vm.prank(RISK_COUNCIL);
+    GHO_AAVE_STEWARD.updateGhoBorrowRate(
+      currentRates.optimalUsageRatio,
+      currentRates.baseVariableBorrowRate + 1,
+      currentRates.variableRateSlope1,
+      currentRates.variableRateSlope2
+    );
+    assertEq(_getGhoBorrowRate(), currentRates.baseVariableBorrowRate + 1);
+  }
+
+  function testGhoBucketStewardUpdateFacilitatorBucketCapacity() public {
+    (uint256 currentBucketCapacity, ) = GhoToken(GHO_TOKEN).getFacilitatorBucket(
+      address(GHO_ATOKEN)
+    );
+    vm.prank(RISK_COUNCIL);
+    uint128 newBucketCapacity = uint128(currentBucketCapacity) + 1;
+    GHO_BUCKET_STEWARD.updateFacilitatorBucketCapacity(address(GHO_ATOKEN), newBucketCapacity);
+    (uint256 capacity, ) = GhoToken(GHO_TOKEN).getFacilitatorBucket(address(GHO_ATOKEN));
+    assertEq(newBucketCapacity, capacity);
+  }
+
+  function testGhoBucketStewardSetControlledFacilitatorAdd() public {
+    address[] memory oldControlledFacilitators = GHO_BUCKET_STEWARD.getControlledFacilitators();
+    address[] memory newGsmList = new address[](1);
+    address gho_gsm_4626 = makeAddr('gho_gsm_4626');
+    newGsmList[0] = gho_gsm_4626;
+    vm.prank(OWNER);
+    GHO_BUCKET_STEWARD.setControlledFacilitator(newGsmList, true);
+    address[] memory newControlledFacilitators = GHO_BUCKET_STEWARD.getControlledFacilitators();
+    assertEq(newControlledFacilitators.length, oldControlledFacilitators.length + 1);
+    assertTrue(_contains(newControlledFacilitators, gho_gsm_4626));
```

Comment:
this indicates we need to add an `isControlledFacilitator` getter

Reply:
try to remove as well pls, in the same test

Reply:
Done